Traceback (most recent call last):
  File "/nix/store/qrkgxqbbr00qjik2rfqdw3z8a062s3ls-python3.8-jupyter-cache-0.4.2/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/nix/store/ria264d36cmhxik90bmzwi12s7nll1y3-python3.8-nbclient-0.5.3/lib/python3.8/site-packages/nbclient/client.py", line 1112, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/nix/store/ria264d36cmhxik90bmzwi12s7nll1y3-python3.8-nbclient-0.5.3/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/nix/store/ria264d36cmhxik90bmzwi12s7nll1y3-python3.8-nbclient-0.5.3/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/nix/store/4s0h5aawbap3xhldxhcijvl26751qrjr-python3-3.8.9/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/nix/store/ria264d36cmhxik90bmzwi12s7nll1y3-python3.8-nbclient-0.5.3/lib/python3.8/site-packages/nbclient/client.py", line 553, in async_execute
    await self.async_execute_cell(
  File "/nix/store/ria264d36cmhxik90bmzwi12s7nll1y3-python3.8-nbclient-0.5.3/lib/python3.8/site-packages/nbclient/client.py", line 857, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/nix/store/ria264d36cmhxik90bmzwi12s7nll1y3-python3.8-nbclient-0.5.3/lib/python3.8/site-packages/nbclient/client.py", line 760, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import babypandas as bpd
import matplotlib.pyplot as plt

birthdays = bpd.read_csv('../../data/us_total_births_2000-2014_no_leaps.csv')
birthdays.plot(kind='bar', x='day_of_year', y='births', width=1, legend=False)
plt.xlabel('Day of year')
plt.xticks(range(0, 366, 10), range(0, 366, 10))
plt.ylabel('Frequency')
plt.title('Sample of U.S. Total Births per Day, 2000-2014')
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[0;32m<ipython-input-1-c6959d83ce3d>[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0;32mimport[0m [0mbabypandas[0m [0;32mas[0m [0mbpd[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m [0;32mimport[0m [0mmatplotlib[0m[0;34m.[0m[0mpyplot[0m [0;32mas[0m [0mplt[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      3[0m [0;34m[0m[0m
[1;32m      4[0m [0mbirthdays[0m [0;34m=[0m [0mbpd[0m[0;34m.[0m[0mread_csv[0m[0;34m([0m[0;34m'../../data/us_total_births_2000-2014_no_leaps.csv'[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m [0mbirthdays[0m[0;34m.[0m[0mplot[0m[0;34m([0m[0mkind[0m[0;34m=[0m[0;34m'bar'[0m[0;34m,[0m [0mx[0m[0;34m=[0m[0;34m'day_of_year'[0m[0;34m,[0m [0my[0m[0;34m=[0m[0;34m'births'[0m[0;34m,[0m [0mwidth[0m[0;34m=[0m[0;36m1[0m[0;34m,[0m [0mlegend[0m[0;34m=[0m[0;32mFalse[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;31mModuleNotFoundError[0m: No module named 'matplotlib'
ModuleNotFoundError: No module named 'matplotlib'

